HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -470,26 +473,26 @@ def add_mNrmStE(self, solution):
         # All combinations of c and m that yield E[PermGroFac PermShkVal mNext] = mNow
         # https://econ-ark.github.io/BufferStockTheory/#The-Individual-Steady-State
 
-        PF_RNrm = self.Rfree/self.PermGroFac
+        PF_RNrm = self.Rfree / self.PermGroFac
```

Comment:
In general ... the reformatting across the library makes it harder to figure out the changes made in this specific PR.

You might consider making a separate PR with just these formatting changes, which could be merged easily.

That would help reviewers focus on the substantive changes in code.